.github/workflows/tests.yml

```diff
@@ -62,6 +62,7 @@ jobs:
           pip install wheel
           pip install --upgrade setuptools
           pip install --editable ".[test,ml,medical,dev,ocv]"
+          pip install albumentations
```

Comment:
Perhaps we do not want to deploy this "quck-fix" to master? Not sure why albumentations are not installed otherwise..

Reply:
That worked great,thanks!